ontology.ttl

```diff
@@ -77,42 +102,148 @@
   rdfs:comment "A File associated with a Publication" .
 
 # Roles
-:CreatorRole a rdfs:Class ;
-  rdfs:comment "The base class for all creator roles" ;
+
+:Advisor a rdfs:Class ;
+  rdfs:comment "A contribution in a supervisory capacity in relation to a resource" ;
+  rdfs:subClassOf :Role .
+
+:Creator a rdfs:Class ;
+  rdfs:comment "A contribution in that plays a major role in the creation of a resource" ;
+  rdfs:subClassOf :Role .
+
+:Editor a rdfs:Class ;
+  rdfs:comment "A contribution from someone who oversees details related to the publication format of a resource" ;
+  rdfs:subClassOf :Role .
+
+:Illustrator a rdfs:Class ;
+  rdfs:comment "A contribution of illustrations, images and/or figures for a resource" ;
+  rdfs:subClassOf :Role .
+
+:Other a rdfs:Class ;
+  rdfs:comment "A contribution of a kind not covered by the other roles" ;
+  rdfs:subClassOf :Role .
+
+# This is not a contribution, it is something else
+:ContactPerson a rdfs:Class ;
+  rdfs:comment "A contribution of being the person who may be contacted related to a resource" ;
+  rdfs:subClassOf :Role .
+
+# This does not fit entirely with our conceptualisation of Contributor since it may also be an institution in Datacite
+:DataCollector a rdfs:Class ;
+  rdfs:comment "Person/institution responsible for finding, gathering/collecting data under the guidelines of the author(s) or Principal Investigator" ;
+  rdfs:subClassOf :Role .
+
+:DataCurator a rdfs:Class ;
+  rdfs:comment "A contribution in reviewing, enhancing, cleaning, or standardizing metadata and associated data submitted for storage, use, and maintenance within a data centre or repository" ;
+  rdfs:subClassOf :Role .
+
+# Again only partial overlap
+:DataManager a rdfs:Class ;
+  rdfs:comment "Person (or organisation with a staff of data managers, such as a data centre) responsible for maintaining the finished resource" ;
+  rdfs:subClassOf :Role .
+
+# This is in essence should be covered by the NVA owner institution and repeats this value unnecessarily
+:Distributor a rdfs:Class ;
+  rdfs:comment "Institution tasked with responsibility to generate/disseminate copies of the resource in either electronic or print form" ;
+  rdfs:subClassOf :Role .
+
+# This maps to grant information on the publication and repeats the information there
+:Funder a rdfs:Class ;
+  rdfs:comment "A contribution of funding from an organization funding the production of the resource" ;
+  rdfs:subClassOf :Role .
+
+# This is always Unit, is it not, covered by our contract with the customer institution.
+:HostingInstitution a rdfs:Class ;
```

Comment:
Unit is always hosting institution for the metadata record (in NVA), but the content can be stored somewhere else. LIke Uninett Sigma, Wiley, NSD or TSD.